client/src/pages/Admin.jsx

```diff
@@ -173,17 +177,24 @@ const FileDownloader = ({ filename, endpoint, text }) => {
 
     getLastImportedFile(endpoint)
       .then((url) => {
+        setError(false);
         link.href = url;
         link.click();
         link.href = "none";
       })
-      .catch((error) => console.error("Can't download the file", error));
+      .catch((error) => {
+        console.error(error);
+        setError(true);
+      });
   }
 
   return (
-    <a href={"none"} download={filename} onClick={fetchAndOpenFile}>
-      {text}
-    </a>
+    <>
+      {isError && <FloatingError message="Impossible de télécharger le fichier" />}
+      <a href={"none"} download={filename} onClick={fetchAndOpenFile}>
```

Comment:
```suggestion
      <a href="none" download={filename} onClick={fetchAndOpenFile}>
```
If "none" is a string you can just do that, but that's a detail